worker/socialPoster.js

```diff
@@ -92,21 +92,29 @@ async function getHottestItem ({ models }) {
   return item[0]
 }
 
-async function itemToMessage ({ item }) {
+async function itemToMessage ({ item, postAuthorNostrProfile }) {
   return `${item.title}
 
-by ${item.userName} in ~${item.subName}
+by ${item.userName}${postAuthorNostrProfile ? `, nostr profile: nostr:${postAuthorNostrProfile}` : ''} in ~${item.subName}
 ${numWithUnits(msatsToSats(item.msats), { abbreviate: false })} and ${numWithUnits(item.ncomments, { abbreviate: false, unitSingular: 'comment', unitPlural: 'comments' })} so far
 
 https://stacker.news/items/${item.id}`
 }
 
 export async function postToSocial ({ models }) {
   const item = await getHottestItem({ models })
-  if (item) {
-    const message = await itemToMessage({ item })
-    console.log('Message:', message)
-    await postToTwitter({ message })
-    await postToNostr({ message })
-  }
+  if (!item) return
+
+  const postAuthor = await models.user.findUnique({
+    where: { id: item.userId, hideNostr: false },
+    select: { nostrPubkey: true }
+  })
+  const postAuthorNostrProfile = postAuthor?.nostrPubkey
+    ? getNostrProfile(postAuthor?.nostrPubkey)
+    : null
+
+  const message = await itemToMessage({ item, postAuthorNostrProfile })
+  console.log('Message:', message)
+  await postToTwitter({ message })
```

Comment:
This includes the nostr mention in the twitter message. This will look bad on twitter.